core/src/avm1/globals.rs

```diff
@@ -129,7 +129,7 @@ pub fn parse_int<'gc>(
         }
 
         // Auto-detect hexadecimal prefix and strip it.
-        // Emulate bug: the prefix is stripped irregardless of the radix.
+        // Emulate bug: the prefix is stripped regardless of the radix.
```

Comment:
I was interested in this change and found this:

> Irregardless is a word sometimes used in place of regardless or irrespective, which has caused controversy since the early twentieth century, [...] Most dictionaries list it as non-standard or incorrect usage, and recommend that "regardless" should be used instead.

https://en.wikipedia.org/wiki/Irregardless